cognite_toolkit/_builtin_modules/inrobot/default.config.yaml

```diff
@@ -1,3 +1,3 @@
-first_root_asset_external_id: <change_me>
-first_location: <change_me>
+first_root_asset_external_id: my_root_asset
+first_location: my_location
```

Comment:
These are used as part of a `space` name, thus changed the defaults to not include `<>`.